### PR TITLE
Adding blueprint for node-uuid

### PR DESCRIPTION
**Pull request by dalvarez**

### CODE REVIEW

File: blueprints/frontend-shared-libs/index.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = {
+  name: 'frontend-shared-libs',
+  
+  included: function(app) {
+      this._super.included(app);
```
Comment (zhujy8833) [inline blueprints/frontend-shared-libs/index.js]: there are some extra indentions

**Comment by dalvarez** [comment]
@zhujy8833 hehe - yeah it still doesnt work though. gonna talk to alex to see how to fix it

File: blueprints/frontend-shared-libs/index.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+module.exports = {
+  name: 'frontend-shared-libs',
+    
+  afterInstall: function() {
```
Comment (twokul) [inline blueprints/frontend-shared-libs/index.js]: should happen on `beforeInstall`

Comment (twokul) [inline blueprints/frontend-shared-libs/index.js]: also I think it should be `return this.addBowerPackageToProject('node-uuid', '~1.4.3');`

**Comment by dalvarez** [comment]
@twokul changed. I didnt realize that it had to return in that function since I didnt see it in the frontend-shared-configs afterInstall()

**Pull request merged**